ambry-server/src/main/java/com/github/ambry/server/StatsManager.java

```diff
@@ -108,6 +112,13 @@
       logger.info("Stats Manager's state change listener registered!");
     }
     this.accountStatsMySqlStore = accountStatsMySqlStore;
+    Function<List<String>, List<Short>> convertAccountNamesToIds = names -> names.stream()
+        .map(name -> accountService.getAccountByName(name))
```

Comment:
Just verifying that account service is already initialized (cache populated) before this constructor.  (Probably is.)

Reply:
yep, the caching part should happen in the AccountService's constructor.